core/src/main/java/bisq/core/payment/payload/PaymentMethod.java

```diff
@@ -74,6 +74,7 @@
     public static final String FASTER_PAYMENTS_ID = "FASTER_PAYMENTS";
     public static final String NATIONAL_BANK_ID = "NATIONAL_BANK";
     public static final String JAPAN_BANK_ID = "JAPAN_BANK";
+    public static final String AUSTRALIA_PAY_ID_ID = "AUSTRALIA_PAYID";
```

Comment:
I think PayID should be considered one word. That seems to be how it's used at https://payid.com.au/

Their capitalization is kind of camel case, but I think it would make more sense to treat it as a single word. It's also not an id within bisq, just part of the name of this payment method so it would make things less confusing, like
```suggestion
    public static final String AUSTRALIA_PAYID_ID = "AUSTRALIA_PAYID";
```